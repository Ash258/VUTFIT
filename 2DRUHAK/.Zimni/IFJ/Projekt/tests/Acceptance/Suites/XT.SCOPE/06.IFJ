' Declaration in nested flow control statements

scope
	dim i1 as integer = 10
	dim i2 as integer = 10

	do while i1 > 0
		do while i2 > 0
			dim a as integer = 42
			a = a + i1 + i2
			print a;
			i2 = i2 - 1
		loop
		print !"\n";
		i1 = i1 - 1
		i2 = 10
	loop
end scope
