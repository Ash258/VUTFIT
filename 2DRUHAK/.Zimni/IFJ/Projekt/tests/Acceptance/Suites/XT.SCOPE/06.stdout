 62 61 60 59 58 57 56 55 54 53
 61 60 59 58 57 56 55 54 53 52
 60 59 58 57 56 55 54 53 52 51
 59 58 57 56 55 54 53 52 51 50
 58 57 56 55 54 53 52 51 50 49
 57 56 55 54 53 52 51 50 49 48
 56 55 54 53 52 51 50 49 48 47
 55 54 53 52 51 50 49 48 47 46
 54 53 52 51 50 49 48 47 46 45
 53 52 51 50 49 48 47 46 45 44
